alchemy/src/util/telemetry.ts

```diff
@@ -0,0 +1,339 @@
+import envPaths from "env-paths";
+import { exec } from "node:child_process";
+import { mkdir, readFile, writeFile } from "node:fs/promises";
+import os from "node:os";
+import { join } from "node:path";
+import pkg from "../../../package.json" with { type: "json" };
+import type { Phase } from "../alchemy.ts";
+import { Scope } from "../scope.ts";
+import { logger } from "./logger.ts";
+import { memoize } from "./memoize.ts";
+
+export const CONFIG_DIR = envPaths("alchemy", { suffix: "" }).config;
```

Comment:
should we move to `~/.alchemy`?

Reply:
only question is how does it work in CI/CD environments?

Reply:
works the same way, where we still get a user id for it, its just different every time.

we have the `isCI` flag to work around that